.github/workflows/ci-e2e-no-metrics-tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Continuous Integration (E2E Testing Checks without metrics database)
+
+on:
+  workflow_call:
+jobs:
+  e2e-no-metrics-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo from current commit
+        uses: actions/checkout@v3
+      - name: set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: "1.21"
+          check-latest: true
+          cache: true
+      - name: pull pre-built images
+        run: sudo docker compose -f ci.docker-compose-no-metrics.yml pull
+      # In this step, this action saves a list of existing images,
+      # the cache is created without them in the post run.
+      # It also restores the cache if it exists.
+      # TODO(yevhenii): this step failed with "No space left on device" error, debug it and enable back
+      # - name: cache docker images
+      #   uses: satackey/action-docker-layer-caching@v0.0.11
+        # Ignore the failure of a step and avoid terminating the job.
```

Comment:
is this still relevant?

Reply:
This step was added by Levi, but it didn't work, so I commented it out. I'm okay if we want to remove it.

Reply:
Removed